### PR TITLE
Eirini native logging with eirini-loggregator-bridge

### PR DESCRIPTION
**Pull request by mudler**
Hi! 

This PR is an example of how to integrate eirini-loggregator-bridge and switch it in place of fluentd. We tested this locally and seems to work.

### CODE REVIEW

File: helm/eirini/templates/eirini-loggregator-bridge.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - key: doppler-cert-key
+        path: doppler.key
+      - key: internal-ca-cert
+        path: doppler.ca
+  initContainers:
+  - name: create-eirini-loggregator-bridge-config
+    volumeMounts:
+    - name: config
+      mountPath: /config
+    image: splatform/eirini-loggregator-bridge
```
Comment (jimmykarily) [inline helm/eirini/templates/eirini-loggregator-bridge.yaml]: https://github.com/SUSE/eirini-loggregator-bridge/blob/master/Dockerfile . This will need to be built in some pipeline.

**Comment by jimmykarily** [comment]
The code for the loggregator bridge component lives here: https://github.com/SUSE/eirini-loggregator-bridge

**Comment by jimmykarily** [comment]
We did some testing with up to 100 applications on a kind cluster (single node cluster). We need to test this with bigger clusters since we then hit the maximum number of pods per node limit on Kubernetes.  We are planning to test it more. Your feedback is also welcome.

Regarding scaling, right now it is not possible to have more than one instance of this component. On the other hand, afaict fluentd also doesn't scale, other than the splitting of one instance per node (daemonsets). We could easily achieve the same if each instance of this component monitored only pods from a specific node, or something like that. This is not implemented yet and it wouldn't be exactly what we call horizontal scaling but rather partitioning or sharding. We are planning to try the daemonset option soon before we start implementing something more complex.
